discord_bots/cogs/admin.py

```diff
@@ -0,0 +1,692 @@
+                    session.query(QueueWaitlist)
+                    .filter(
+                        QueueWaitlist.queue_id == queue.id,
+                    )
+                    .first()
+                )
+                if queue_waitlist:
+                    session.query(QueueWaitlistPlayer).filter(
+                        QueueWaitlistPlayer.player_id == member.id,
+                        QueueWaitlistPlayer.queue_waitlist_id == queue_waitlist.id,
+                    ).delete()
+
+            queue_statuses = []
+            queue: Queue
+            for queue in session.query(Queue).order_by(Queue.created_at.asc()).all():  # type: ignore
+                queue_players = (
+                    session.query(QueuePlayer)
+                    .filter(QueuePlayer.queue_id == queue.id)
+                    .all()
+                )
+                queue_statuses.append(
+                    f"{queue.name} [{len(queue_players)}/{queue.size}]"
+                )
+            session.commit()
+
+        await interaction.response.send_message(
+            embed=Embed(
+                title=f"{escape_markdown(member.name)} removed from: {', '.join([queue.name for queue in queues])}",
+                description=" ".join(queue_statuses),
+                colour=Colour.green(),
+            )
+        )
+
+    @group.command(name="editcommand", description="Edit a custom command")
+    @app_commands.check(is_admin_app_command)
+    @app_commands.check(is_command_channel)
+    @app_commands.describe(
+        name="Name of existing custom command", output="New command output"
+    )
+    async def editcommand(self, interaction: Interaction, name: str, *, output: str):
+        session: SQLAlchemySession
+        with Session() as session:
+            exists = (
+                session.query(CustomCommand).filter(CustomCommand.name == name).first()
+            )
+            if exists is None:
+                await interaction.response.send_message(
+                    embed=Embed(
+                        description="Could not find a command with that name",
+                        colour=Colour.red(),
+                    ),
+                    ephemeral=True,
+                )
+                return
+
+            exists.output = output
+            session.commit()
+
+        await interaction.response.send_message(
+            embed=Embed(
+                description=f"Command `{name}` updated",
+                colour=Colour.green(),
+            )
+        )
+
+    @group.command(
+        name="editgamewinner", description="Edit the winner of a finished game"
+    )
+    @app_commands.check(is_admin_app_command)
+    @app_commands.check(is_command_channel)
+    @app_commands.describe(game_id="Finished game id", outcome="<tie|be|ds>")
+    async def editgamewinner(
+        self,
+        interaction: Interaction,
+        game_id: str,
+        outcome: Literal["Tie", "BE", "DS"],
```

Comment:
For the future, we may want to move away from the BE/DS convention here, and just do `team0, team1` or `team1, team2`.

Reply:
Added a `TODO`